### ISSUE TITLE
Document Package Installation & Use

### ISSUE THREAD

**Issue by OlinGRAVL** [olinrobotics/state_controller#1]
Add README.md describing package, how it works (diagram), and how to use it

**Comment by nathanestill** [comment]
I maybe did this, but you should definitely check it.

**Comment by ConnorNovak** [comment]
Yeah, we need to update this for the blade attribute command system. Probably should clean that up first, make a new image, etc. The two commands should probably be rolled into a single message, but that will be a new package installation.
